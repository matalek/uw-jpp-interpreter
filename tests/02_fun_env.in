int a;

int p(int n) {
	bool a;
	a = false;
	return a;
}

int main(){
	a = 1;
	print a;
	print p(0);
	print a;
	return;
}
