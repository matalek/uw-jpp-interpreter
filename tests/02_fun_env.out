1
false
1

